generate/input/descriptor.json

```diff
@@ -992,6 +1021,13 @@
               "isReturn": true
             }
           },
+          "isAsync": true,
+          "return": {
+            "isErrorCode": true
+          }
+        },
+        "git_index_conflict_remove": {
```

Comment:
If you want to be pedantic about keeping things sorted alphabetically, this one belongs a few entries down